mobile/mysterium/payment_orders.go

```diff
@@ -145,12 +145,13 @@ func (mb *MobileNode) CreatePaymentGatewayOrder(req *CreatePaymentGatewayOrderRe
 	}
 
 	order, err := mb.pilvytisOrderIssuer.CreatePaymentGatewayOrder(
-		identity.FromAddress(req.IdentityAddress),
-		req.Gateway,
-		req.MystAmount,
-		req.PayCurrency,
-		req.Country,
-		req.GatewayCallerData,
+		pilvytis.GatewayOrderRequest{Identity: identity.FromAddress(req.IdentityAddress),
```

Comment:
This should now probably send some projectID no?

Reply:
projectID is defaulted to `mysteriumvpn` which is reserved for desktop and mobile :)